src/test/java/com/redhat/devtools/lsp4ij/fixtures/LSPCodeInsightFixtureTestCase.java

```diff
@@ -37,6 +38,11 @@ public LSPCodeInsightFixtureTestCase(String... fileNamePatterns) {
         this.fileNamePatterns = fileNamePatterns;
     }
 
+    protected LSPCodeInsightFixtureTestCase(@NotNull String fileNamePattern, @NotNull String languageId) {

```

Comment:
Since filename matchers aren't registered retained in the language server definition unless there's a language ID, I added a convenience constructor for a single file name pattern and a language ID.